### ISSUE TITLE
IAppsFlyerConversionData is private

### ISSUE THREAD

**Issue by revolt3r** [AppsFlyerSDK/appsflyer-unity-plugin#14]
It's impossible to implement the interface because it's private.

**Comment by wesfieldj** [comment]
@revolt3r Can you please provide some more information/code snippet? 

We are able to implement the IAppsFlyerConversionData with this sample code:
```c#
using System.Collections;
using System.Collections.Generic;
using AppsFlyerSDK;
using UnityEngine;

public class AFObject : MonoBehaviour, IAppsFlyerConversionData
{
    void Start()
    {
        AppsFlyer.setIsDebug(true);
        AppsFlyer.initSDK("K2****9", "416***5", this);
        AppsFlyer.startSDK();
    }
    
    public void onConversionDataSuccess(string conversionData)
    {
        print(conversionData);
    }

    public void onConversionDataFail(string error)
    {
    }

    public void onAppOpenAttribution(string attributionData)
    {
    }

    public void onAppOpenAttributionFailure(string error)
    {
    }
}
```

**Comment by revolt3r** [comment]
I think I just found the issue. We are keeping Plugins in a separate assembly from our game code. When AppsFlyer is in a different assembly, you are not able to implement the private interface.
I think the same thing is going to happen once we implement AppsFlyer as a package

**Comment by wesfieldj** [comment]
@revolt3r We would like to get this solved for you and add a fix to the next release.

Will adding a public modifier solve this? 

`public interface IAppsFlyerConversionData`

Can you try to add public modifiers in IAppsFlyerConversionData and check if that solves the issue?

Thanks!

**Comment by revolt3r** [comment]
Yes, it solved it, thanks!

**Comment by wesfieldj** [comment]
Great! So we will add it to the next release.

**Comment by aperevalov** [comment]
Please also make public interfaces IAppsFlyerUserInvite and IAppsFlyerValidateReceipt.
There will be the same issue if AppsFlyer will be in separate assembly definition. by default its internal and not visible to other assemblies.

**Comment by wesfieldj** [comment]
Fixed in v5.3.1
